web-client/public/js/upload.js

```diff
@@ -1,3 +1,12 @@
+// TODO Likely a temporary location, while things are being moved to their "true" homes.
+//      But placed here for now so that the true MVC cycle of grnState, updateApp, and the
+//      controller code installed by setupHandlers can access them.
+export const uploadState = {
+    currentNetwork: null,
+    sliders: null,
+    nodeColoring: null
+};
```

Comment:
This is the workaround for dealing with dependencies on variables that haven't been cleaned up yet. It does the job without requiring large changes to the code. Notice the additional lines below where we relay the values to this "holding object" as they are assigned or initialized.

Reply:
As the transition proceeds, this object will probably grow a bit more. But presumably as things get more consolidated, it will then start contracting again as its code/values find their rightful place either in `grnState`, `updateApp`, or `setupHandlers`. Eventually it should disappear entirely.